### PR TITLE
Feature fix map creation

### PR DESCRIPTION
**Pull request by oscar134**
Remove description minimum length validation, add real author to the route, searching bar not yet.

### CODE REVIEW

**Comment by oscar134** [comment]
Not finished

File: src/model/Util.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const auth = require('solid-auth-client');
+
+class UserDetails {
+
+     static async getName(){
+        var session = await auth.currentSession();
+        var username = session.webId;
+        username = username.replace('https://', '');
+        username = username.replace('.solid.community/profile/card#me', '');
```
Comment (jesQM) [inline src/model/Util.js:9]: Be wary, this wont work if the user is using inrupt provider.

However, I believe this replace can be replaced by a query using ldflex:
https://solid.github.io/ldflex-playground/#%5B'https%3A%2F%2Fuo263624.solid.community%2Fprofile%2Fcard%23me'%5D.name

Comment (oscar134) [reply src/model/Util.js:9]: I simply refactored the one we were using in the home view. Good thing is that now we only have to change it in one place.

**Comment by oscar134** [comment]
This is impossible to test. Whenever I uses promises in tests, it fails something about JSON!!!!!!!!!!!

**Comment by oscar134** [comment]
This branch corresponds to the Search Bar Issue (#143). Is available to be reviewed. The problem that @jesQM mentions above is not solved, as @nicomencia is working on it.

**Comment by nicomencia** [comment]
There is still a bug. When the translation is called the username disapears.

**Pull request merged**